clap-utils/src/keypair.rs

```diff
@@ -431,6 +444,48 @@ pub fn prompt_passphrase(prompt: &str) -> Result<String, Box<dyn error::Error>>
     Ok(passphrase)
 }
 
+/// Parses a path into a SignerSource and returns a Keypair for supporting SignerSourceKinds
+pub fn keypair_from_path(
```

Comment:
Technically this can be done using `signer_from_path` and downcasting from `dyn Signer`. However, that requires implementing an `Any` method for every signer; a bit of churn. Also I thought this function might be useful in other places in the future.

Reply:
I prefer this to runtime reflection!